readme.md

```diff
@@ -764,6 +764,7 @@
 - [Playwright](https://github.com/mxschmitt/awesome-playwright#readme) - Node.js library to automate Chromium, Firefox and WebKit with a single API.
 - [Quality Assurance Roadmap](https://github.com/fityanos/awesome-quality-assurance-roadmap#readme) - How to start & build a career in software testing.
 - [Gatling](https://github.com/aliesbelik/awesome-gatling#readme) - Open-source load and performance testing framework based on Scala, Akka, and Netty.
+- [Locust](https://github.com/aliesbelik/awesome-locust#readme) - Open-source Python-based performance testing tool.
```

Comment:
Your PR has passed in the `awesome-lint` checker

```sh
❯ npx awesome-lint https://github.com/aliesbelik/awesome-locust
✔ Linting
```

All good to be accepted.